extensions/wikia/Answers/HomePageList.php

```diff
@@ -277,4 +277,19 @@ function _homepage_questions($key, $title, $ignore_cache = false) {
 		wfProfileOut(__METHOD__);
 		return $html;
 	}
+
+	/**
+	 * @param string $pageTitle
+	 * @param string $text
+	 * @return string
+	 */
+	private function createLinkListElement($pageTitle, $text): string
```

Comment:
Let's use [MW coding conventions](https://www.mediawiki.org/wiki/CC/PHP) - there's also an [XML configuration snippet](https://www.mediawiki.org/wiki/JetBrains_IDEs#MediaWiki_code_style) for IDEA